libs/sdk/src/lib/helpers/get-token-address.ts

```diff
@@ -0,0 +1,8 @@
+import { getPublicKey } from '@kin-kinetic/solana'
+import { getAssociatedTokenAddress } from '@solana/spl-token'
+
+export async function getTokenAddress({ account, mint }: { account: string; mint: string }): Promise<string> {
```

Comment:
Does this need to be async? Maybe getAssociatedtokenAddress is async in JS? It shouldn't be though, it's just doing math.